src/v2/oracles/GdaiPriceProvider.sol

```diff
@@ -0,0 +1,64 @@
+/******************************************************* 
+NOTE: Development in progress by JG. Reached functional milestone; Live VST data is accessible. 
```

Comment:
is this note still relevant?

Reply:
It's not - I will remove this

Have an unrelated Q - timestamps are being emitted in the genericController events. Are you happy for me to remove the timestamp as they can be implictly fetched from the block?

Reply:
Indeed that makes a lot of sense, however we are the parameter in the subgraph. So for Standart V2 Controller we can leave them in, however for any other we can remvoe them, as we anyways have new ABI/ need new handlers on the subgraph.